ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -269,7 +268,6 @@ objects:
       - name: PULL_SECRET_PATH
         value: /etc/openshift-installer/pull-secret
       - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
-        value: registry.svc.ci.openshift.org/ocp-s390x/release-s390x:4.2
```

Comment:
This will be supplied by the job yaml, which will pass it into the env vars, which in turn will supply it to the container.
Job variable -> MULTIARCH_RELEASE_IMAGE_LATEST (value = registry.svc.ci.openshift.org/ocp-s390x/release-s390x:4.2)
Template env variable -> MULTIARCH_RELEASE_IMAGE_LATEST (required, no default value set)
Container variable ->  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE (value = ${MULTIARCH_RELEASE_IMAGE_LATEST})